packages/docs-components/src/knobs/useSelectKnob.ts

```diff
@@ -1,11 +1,19 @@
 import { UseKnobOptions } from './types'
 import useKnob from './useKnob'
 
-const useSelectKnob = <T extends string>(options: UseKnobOptions<T>) =>
-  useKnob<T>({
+type UseSelectKnobOptions<T extends string> = UseKnobOptions<T> & {
+  allowsNone?: boolean
```

Comment:
this is strange. can't we just rely on `undefined` as value?

Reply:
```tsx
const [values] = useSelectKnob({ allowsNone: true, values: ['a', 'b'] })
const [values] = useSelectKnob({ values: [undefined, 'b'] })
```

Are you sure that second one is more obvious? 🤔

Reply:
I would vote for `const [values] = useSelectKnob({ allowsNone: true, values: ['a', 'b'] })` It is more obvious what will be rendered, but that's just me :)